### PR TITLE
Merge release-3.8.7 into master

### PR DESCRIPTION
**Pull request by thachmai**

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -19,22 +19,41 @@ Release Date: 2018-10-01
 
 New Features
 ~~~~~~~~~~~~
-* UForge administrator can manage distributions and repositories from the user interface
+* UForge adminstrator can now register and manage software repositories and operating systems from the user interface
```
Comment (Numerinico) [inline changelog.rst]: typo "administrator"

File: changelog.rst [hunk 2]
```diff
@@ -19,22 +19,41 @@ Release Date: 2018-10-01
...
+* UForge command line tool option ``--officialySupported`` for command ``org repo create`` is deprecated. Please Use ``--coreRepository`` instead
+
+Bug Fixes
+~~~~~~~~~
+* 8341 Fixed issue allowing japanese characters to be used in the description field for an OS Profile
+* 8934 Fixed issue to allow a user to correctly delete a pinned package in the OS profile of a template
+* 8936 Fixed validation tooltips where backslashes are actually not supported
+* 8940 Fixed unclear error message when editing a software component bootscript used in an ongoing generation
+* 10708 Fixed issue when generating an ISO image from a scan of a live system with CentOS installed
+* 10822	Fixed issue generating Ubuntu 10.04 with the latest ``debootstrap`` package
```
Comment (Numerinico) [inline changelog.rst]: empty space ?

File: changelog.rst [hunk 3]
```diff
@@ -19,22 +19,41 @@ Release Date: 2018-10-01
...
+* 11096	Fixed the unpinning of a package from failing in the UI
```
Comment (Numerinico) [inline changelog.rst]: same

File: changelog.rst [hunk 4]
```diff
@@ -19,22 +19,41 @@ Release Date: 2018-10-01
...
+* 11201 Fixed arrow buttons from expanding in Distribution > OS Profile view in the UI
+* 11226 Cannot log in to a CentOS AWS instance with SSH key pair set by AWS
+* 11292 Replaced ``vssadmin.exe`` with ``diskshadow.exe`` to properly flush Windows registry during migration process
+* 11349 Fix to display tenant name in the details of a published image
```
Comment (Numerinico) [inline changelog.rst]: Should we specify it is for OpenStack ?

File: end-user/en/source/pages/generating/supported-image-type.rst [hunk 5]
```diff
@@ -83,7 +83,8 @@ With UForge you can create machine images in the following formats.
 |                                     | CloudStack Citrix Xen (VHD)                           |
 |                                     | CloudStack KVM (QCOW2)                                |
 +-------------------------------------+-------------------------------------------------------+
-| Fujitsu K5                          | 32 bit OSes are not supported                         |
+| Fujitsu K5                          | 32 bit OSes are not supported.                        |
+|                                     | For Windows, Evaluation Editions are not supported    |
```
Comment (Numerinico) [inline end-user/en/source/pages/generating/supported-image-type.rst]: "." or not dot, maybe we should be consistent ?

File: changelog.rst (hunk 3 above)
Comment (thachmai) [reply changelog.rst]: done

File: changelog.rst (hunk 2 above)
Comment (thachmai) [reply changelog.rst]: done

File: changelog.rst (hunk 1 above)
Comment (thachmai) [reply changelog.rst]: corrected

File: changelog.rst (hunk 4 above)
Comment (thachmai) [reply changelog.rst]: added "for OpenStack"

File: end-user/en/source/pages/generating/supported-image-type.rst (hunk 5 above)
Comment (thachmai) [reply end-user/en/source/pages/generating/supported-image-type.rst]: removed .

**Comment by thachmai** [comment]
Need to cherry-pick https://github.com/usharesoft/appcenter-docs/pull/937/commits/379d757ef0e68ec0953f8402c7a80f7beaed4306 into `3.8.fp7` branch

**Pull request merged**